### PR TITLE
Get Random Value

### PR DESCRIPTION
**Pull request by MohammadMazin**
Returns a random value from the array.

Also caters for edge cases:
1- returns a TypeError using the implemented _throw() function.
2- returns null if length of array is 0.

### CODE REVIEW

**Comment by Hetarth02** [comment]
Thanks for PR. I have left some feedback regarding the changes.

**Comment by MohammadMazin** [comment]
@Hetarth02 I'm still new to this, so I wasn't able to find feedback in this PR. But I noticed the change in package.json. Shall I update that and send an updated PR?

**Comment by Hetarth02** [comment]
@MohammadMazin Sorry for the late response. But you can check out the feedback [here](https://github.com/Hetarth02/js-array-helpers/pull/29#pullrequestreview-1386426381).

And I think there will a `file changed` section where you can see the feedback too.

File: README.md [hunk 1]
```diff
@@ -1,95 +1,6 @@
-# js-array-helpers
```
Comment (MohammadMazin) [inline README.md:1]: .

**Comment by MohammadMazin** [comment]
@Hetarth02 Clicking on the link just redirects me to the conversation tab and I can't seem to find anything on the file changed tab either, no comments are visible there. 😅I can try closing this PR and submitting a new one .

**Comment by Hetarth02** [comment]
Hmm, that's weird. No problem I will accept the PR. No need for a new one.

Thanks for contributing.👍

**Pull request merged**